openapi/services/biblib.yaml

```diff
@@ -548,42 +694,13 @@ biblib-libraries-library_id:
   get:
     summary: View a single library
     parameters:
-      - name: library_id
-        description: Library ID
-        required: true
-        in: path
-        schema:
-          type: string
-      - name: start
-        description: starting document index
-        in: query
-        schema:
-          type: int
-          default: 0
-      - name: rows
-        description: number of documents to return
-        in: query
-        schema:
-          type: int
-          default: 20
-      - name: sort
-        description: document sort order
-        in: query
-        schema:
-          type: string
-          default: 'date desc'
-      - name: fl
-        description: returned fields
-        in: query
-        schema:
-          type: string
-          default: 'bibcode'
-      - name: raw
-        description: returns raw bibcodes from database
-        in: query
-        schema:
-          type: boolean
-          default: false
+      - $ref: '../parameters.yaml#/library_id'
+      - $ref: '../parameters.yaml#/start'
+      - $ref: '../parameters.yaml#/rows_documents'
+      - $ref: '../parameters.yaml#/sort_documents'
+      - $ref: '../parameters.yaml#/fl_documents'
```

Comment:
I think this one is still just `fl` per [this](https://github.com/adsabs/biblib-service/blob/d1c886d67b5b73dfc0367084e940826ee1490132/biblib/views/library_view.py#L224)

Reply:
I think `sort_documents` and is also still just `sort`

Reply:
Oh yeah. I know what happened. Some of these parameters have the same name, but are just slightly different (e.g. have a different description). I was trying to figure what to do, changed a few names and forgot to change it back. I'll change these back right away. And for now, I'll just leave them as they were.

Reply:
Makes sense. If you think it would be better to update the params in `biblib`, that would also be totally fine. The only thing that matters is that the params here and in `biblib` agree.